apps/peertube-runner/src/server/process/shared/process-vod.ts

```diff
@@ -119,11 +119,37 @@ export async function processHLSTranscoding (options: ProcessOptions<RunnerJobVO
       videoFile: videoPath
     }
 
+    if (payload.output.videoFileUrl) {
```

Comment:
Please wrap these `uploadTranscodeResult` calls in `success` method, I think it will be simpler and will require less code